src/main/java/edu/hm/hafner/coverage/FileNode.java

```diff
@@ -646,6 +648,26 @@ public FileNode addFunctionCallCounters(final int lineNumber, final int covered,
         return this;
     }
 
+    /**
+     * Increase the counter based on covered or misses lines.
+     *
+     * @param lineNumber line number
+     * @param covered covered lines
+     * @param missed missed lines
+     * @return fileNode
+     */
+    @CanIgnoreReturnValue
+    public FileNode addCounterIncremental(final int lineNumber, final int covered, final int missed) {
```

Comment:
I don't understand the motivation for this method. As far as I understand the coverage format we only have the information covered or not for each line. I.e., we do not evaluate branch coverage, just line coverage. Line coverage is stored here in these counters with either 1, 0 or 0,1. There is no value > 1.

Reply:
Exactly, we have converted the coverage format generated by GoCov to if a line is coverage or not by using 1 or 0. We are not using any value greater than 1.